content/en/events/kcsna2021/registration.md

```diff
@@ -4,34 +4,13 @@ type: docs
 weight: 2
 ---
 
-### Attend in-person
+As we are not holding the formal Contributor Summit, we are not
+running registration. SIGs and subprojects who choose to hold a mini-event will
+be responsible for any kind of signups they need to manage.
 
-**NOTE:** The Kubernetes Contributor Summit will follow KubeCon’s
+**NOTE:** Any in-person events held by SIGs and subprojects will follow KubeCon’s
 [Health & Safety Guidelines] requiring **ALL** in-person attendees to be fully
 vaccinated against the COVID-19 virus.
 
-To attend in person you must be a member of one of the Kubernetes GitHub Orgs
-(e.g. [kubernetes] or [kubernetes-sigs]) or a sponsored attendee. If you are an
-active contributor but not a member, consider [applying for org membership]!
-
-If you have questions, please see the [faq] or reach out to us at
-community@kubernetes.io.
-
-[Register here]
-
-
-[Register here]: https://www.cvent.com/d/ymq151/4W?ct=f34626fb-147d-492c-81d2-b77ce49232c1
```

Comment:
We will still want to include the reg link, but as an RSVP for the evening social event. TBH, it might make sense to rename the page altogether to be rsvp instead of reg. What do you think?

Reply:
Ahhh, ok. We're using the same link. I like RSVP instead of registration now that it's not exactly a conference. Are we still going to restrict the event to contributors (as in we'll be checking badges somehow at the door)?

Reply:
Ok, updated! Better?